### PR TITLE
note that the CHANGELOG can be found at CHANGES.md for the PR template

### PR DESCRIPTION
**Pull request by raybellwaves**
### Description

While working on my first PR it took me a moment to figure out where the CHANGELOG can found.

This PR makes the location more explicit which hopefully makes PR smoother for other first time contributors.

### Checklist - did you ...

- [ ] Add a CHANGELOG entry if necessary?
- [ ] Add / update tests if necessary?
- [ ] Add new / update outdated documentation?

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -18,7 +18,7 @@
     Tests are required for bugfixes and new features. Documentation changes
     are necessary for formatting and most enhancement changes. -->
 
-- [ ] Add a CHANGELOG entry if necessary?
+- [ ] Add a CHANGELOG entry if necessary (CHANGES.md) ?
```
Comment (JelleZijlstra) [inline .github/PULL_REQUEST_TEMPLATE.md:21]:
```suggestion
- [ ] Add a CHANGES.md entry if necessary?
```
Shorter, and better to use only one term

Comment (raybellwaves) [reply .github/PULL_REQUEST_TEMPLATE.md]: thanks

**Review by isidentical — APPROVED** [review]
Thanks!

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (ichard26) [reply .github/PULL_REQUEST_TEMPLATE.md]: I feel like this assumes people know what `CHANGES.md` references, why not this?
```markdown
- [ ] Add an entry in `CHANGES.md` if necessary?
```

Comment (JelleZijlstra) [reply .github/PULL_REQUEST_TEMPLATE.md]: Sure, that's even better

**Review by ichard26 — APPROVED** [review]
Thank you, this is great. I love removing pain points from the contributing experience :)

**Pull request merged**